src/main/java/com/project/attendancemanagementsystem/controller/AdminController.java

```diff
@@ -0,0 +1,39 @@
+package com.project.attendancemanagementsystem.controller;
+
+import com.project.attendancemanagementsystem.model.StudentRequest;
+import com.project.attendancemanagementsystem.model.StudentResponse;
+import com.project.attendancemanagementsystem.service.AdminService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class AdminController {
+
+    private final AdminService adminService;
+
+    public AdminController(AdminService adminService) {
+        this.adminService = adminService;
+    }
+
+    @PostMapping("/admin")
```

Comment:
And here and in other places the path would be just `"/students"`

Reply:
done